README.md

```diff
@@ -1,41 +1,64 @@
-js-to-fhirpath is a syntax parser that converts normal mathematical expressions
+# Math-to-fhirpath
+
+Math-to-fhirpath is a syntax parser that converts basic mathematical expressions
 into FHIRpath notation. 
 
-USAGE
+# Local Installation
+
+```bash
+npm install
+```
+
+To test that math-to-fhirpath is working correctly:
+
+```bash
+npm test
+```
+
+# Usage
+
+```javascript
+import mathtofhirpath
+mathtofhirpath.fhirconvert("(a+b)^2", [a, b]); // returns ($a+$b).power(2)
+mathtofhirpath.fhirconvert("CEILING(a*b)", [a, b]); // returns ($a*$b).ceiling()
+```
+
+***fhirconvert*** is the main function of math-to-fhirpath, which validates and converts an
+inputted expression to fhirpath. fhirconvert will also return null if the expression fails validation.
+
 
-Normal syntax expressions utilize traditional operators and function statements.
+fhirconvert takes in two inputs: the mathematical expression for conversion and 
+an array of usable variables. The format is as follows:
+
+```javascript
+fhirconvert([expression], [vars])
+```
+
+Basic syntax expressions utilize traditional operators and function statements.
 Expressions can be written using variable names, mathematical operators, and various
-functions. 
+functions. The syntax guide is bellow.
+
+# Syntax Guide
+
+***USABLE OPERATORS:*** +, -, *, /, ^, **, !=, !~, >=, <=, =, &&, ||, xor, and, or, implies
 
-In addition to passing in the expression, an array of usable variables must
-also be passed as a parameter.
+***USABLE FUNCTIONS:***
+CEILING(), FLOOR(), ABS(), LOG(), TRUNCATE(), EXP(), SQRT(), LN(), NOT()
+    -Usage: CEILING([expression]), FLOOR([expression]), etc.
```

Comment:
This line and the one under LOG() did not render as a separate line.  (The bullet syntax isn't quite right.)  See https://github.com/aaronzsun/js-to-fhirpath/blob/master/README.md

Reply:
I see you found a way to that work (by adding some sort of whitespace at the end) but while it seems to work on GitHub, the syntax is not quite markdown, and might not work on Bitbucket or GitLab.  See https://daringfireball.net/projects/markdown/syntax#list, which says, "List markers typically start at the left margin, but may be indented by up to three spaces. List markers must be followed by one or more spaces or a tab."

Reply:
Fixed